dotnet/Templates/Microsoft.MacCatalyst.Templates/maccatalyst/csharp/MacCatalystApp1.csproj

```diff
@@ -1,11 +1,56 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net7.0-maccatalyst</TargetFramework>
+    <!--Both architectures:
+    <RuntimeIdentifier>maccatalyst-x64;maccatalyst-arm64</RuntimeIdentifier>-->
+    <!--Only x64. Please note that selecting only arm64 is not permitted.-->
     <RuntimeIdentifier>maccatalyst-x64</RuntimeIdentifier>
```

Comment:
The RuntimeIdentifier(s) property is a bit confusing, so I think we can do something like:

* Don't set any value in the template.
* In a comment, explain that
    * The default is `maccatalyst-x64`, unless `Configuration=Release`, in which case the default is `maccatalyst-x64;maccatalyst-arm64`
    * The plural version (`RuntimeIdentifiers`) must be used when specifying more than one.
    * The App Store won't allow only `maccatalyst-arm64`,  it's either x64 or both.

Reply:
I left a commented out example of <RuntimeIdentifiers>maccatalyst-x64;maccatalyst-arm64</RuntimeIdentifiers> if that's alright.